cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -48,23 +48,23 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: elasticsearch-logging
-  namespace: kube-system
+  namespace: logging
   labels:
     k8s-app: elasticsearch-logging
-    version: v7.4.3
+    version: v7.4.4
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   serviceName: elasticsearch-logging
   replicas: 2
   selector:
     matchLabels:
       k8s-app: elasticsearch-logging
-      version: v7.4.3
+      version: v7.4.4
   template:
     metadata:
       labels:
         k8s-app: elasticsearch-logging
-        version: v7.4.3
+        version: v7.4.4
```

Comment:
- image: quay.io/fluentd_elasticsearch/elasticsearch:v7.4.3
The image here is 7.4.3.
Why the version is changed to v7.4.4 in labels?

Reply:
the label version is for the entire statefulset.   I believe updating the namespace is sufficient to rev the patch version at least.  but the image isn't moving.  

these will get back in-sync with the next minor release of the image.